docs/deploy_private.md

```diff
@@ -8,41 +8,46 @@ If you want to disable public access when deploying the Chat App, you can do so
 
 ## Environment variables controlling private access
 
 1. `AZURE_PUBLIC_NETWORK_ACCESS`: Controls the value of public network access on supported Azure resources. Valid values are 'Enabled' or 'Disabled'.
-  1. When public network access is 'Enabled', Azure resources are open to the internet.
-  1. When public network access is 'Disabled', Azure resources are only accessible over a virtual network.
+    1. When public network access is 'Enabled', Azure resources are open to the internet.
+    1. When public network access is 'Disabled', Azure resources are only accessible over a virtual network.
 1. `AZURE_USE_PRIVATE_ENDPOINT`: Controls deployment of [private endpoints](https://learn.microsoft.com/azure/private-link/private-endpoint-overview) which connect Azure resources to the virtual network.
-  1. When set to 'true', ensures private endpoints are deployed for connectivity even when `AZURE_PUBLIC_NETWORK_ACCESS` is 'Disabled'.
-  1. Note that private endpoints do not make the chat app accessible from the internet. Connections must be initiated from inside the virtual network.
+    1. When set to 'true', ensures private endpoints are deployed for connectivity even when `AZURE_PUBLIC_NETWORK_ACCESS` is 'Disabled'.
+    1. Note that private endpoints do not make the chat app accessible from the internet. Connections must be initiated from inside the virtual network.
 1. `AZURE_PROVISION_VM`: Controls deployment of a [virtual machine](https://learn.microsoft.com/azure/virtual-machines/overview) and [Azure Bastion](https://learn.microsoft.com/azure/bastion/bastion-overview). Azure Bastion allows you to securely connect to the virtual machine, without being connected virtual network. Since the virtual machine is connected to the virtual network, you are able to access the chat app.
-  1. You must set `AZURE_VM_USERNAME` and `AZURE_VM_PASSWORD` to provision the built-in administrator account with the virtual machine so you can log in through Azure Bastion.
-  1. By default, a server version of Windows is used for the VM. If you need to [enroll your device in Microsoft Intune](https://learn.microsoft.com/mem/intune/user-help/enroll-windows-10-device), you should use a desktop version of Windows by setting the following environment variables:
-    * `azd env set AZURE_VM_OS_PUBLISHER MicrosoftWindowsDesktop`
-    * `azd env set AZURE_VM_OS_OFFER Windows-11`
-    * `azd env set AZURE_VM_OS_VERSION win11-23h2-pro`
+    1. You must set `AZURE_VM_USERNAME` and `AZURE_VM_PASSWORD` to provision the built-in administrator account with the virtual machine so you can log in through Azure Bastion.
+    1. By default, a server version of Windows is used for the VM. If you need to [enroll your device in Microsoft Intune](https://learn.microsoft.com/mem/intune/user-help/enroll-windows-10-device), you should use a desktop version of Windows by setting the following environment variables:
+
+      * `azd env set AZURE_VM_OS_PUBLISHER MicrosoftWindowsDesktop`
```

Comment:
This list was actually broken in rendering, which is why I went down the linting path.